package.json

```diff
@@ -46,16 +49,26 @@
 	],
 	"devDependencies": {
 		"@microsoft/api-extractor": "7.40.1",
+		"@types/fs-extra": "11.0.3",
 		"@types/node": "20.11.17",
 		"@typescript-eslint/eslint-plugin": "7.0.1",
 		"@typescript-eslint/parser": "7.0.1",
 		"eslint": "8.56.0",
 		"eslint-plugin-tsdoc": "0.2.17",
+		"fs-extra": "11.1.1",		
 		"glob": "10.3.10",
 		"hi-base32": "0.5.1",
 		"ts-node": "10.9.2",
 		"tslib": "2.6.2",
 		"typedoc": "0.25.8",
 		"typescript": "5.3.3"
+	},
+	"dependencies": {
+		"@types/xml2js": "^0.4.14",
+		"eslint-config-prettier": "^9.1.0",
+		"eslint-plugin-prettier": "^5.1.3",
+		"m3u8-parser": "^7.1.0",
```

Comment:
I understand that some of the new utility functions require these to be non-dev dependencies, but this means that someone using the library for CMCD, or ID3, has to install these packages. I'm not sure what the best solution for this is, `optionalDependecies` perhaps, but I think we might move towards breaking all of the features out into their own npm packages: `@svta/cml-cmcd`, `@svta/cml-id3`, `@svta/cml-cmaf-ham`. Nothing actionable here, just mentioning it for future reference.